plugins/inputs/t128_lte/t128_lte.go

```diff
@@ -256,28 +253,24 @@ func matchNode(nodeName string, content []byte) (string, error) {
 	return "", fmt.Errorf("No matching node names are available in the config %s", content)
```

Comment:
`matchNode` is still wrong. It's just returning the node name which you already had.